src/ForkCMS/Utility/Thumbnails.php

```diff
@@ -157,4 +147,87 @@ public function getFolders(string $inPath, bool $includeSourceFolder = false): a
 
         return array_values($folders);
     }
+
+    /**
+     * Calculate the desired width based on the aspect ratio if needed
+     *
+     * @param int|null $desiredWidth
+     * @param int|null $desiredHeight
+     * @param int $currentWidth
+     * @param int $currentHeight
+     * @return int
+     */
+    private function calculateDesiredWidth(
+        ?int $desiredWidth,
```

Comment:
I don't think it is good practice to have optional arguments at the beginning.

Reply:
This isn't an optional parameter, it just means that it can be null or int.
I could flip the parameters around so they match what we have in the other method

Reply:
😱true that, seems like it was too early for me.

Reply:
Changed it anyway for consistency sake
14c6ad269